proto/perp/v1/state.proto

```diff
@@ -145,89 +145,6 @@ message PrepaidBadDebt {
   ];
 }
 
-message PositionResp {
```

Comment:
Turned `PositionResp` and `LiquidateResp` into Golang structs since they weren't being serialized or passed around anywhere.